File: proyect_rappi/python/Restaurant.py
from Products import Products

class Restaurant(Products):
    type_food = str
    sell_by_date = str
    name_rest = str
    def __init__(self,name,brand,amount,type_food,sell_by_date,name_rest):
        super().__init__(name,brand,amount)
        self.type_food = type_food
        self.sell_by_date = sell_by_date
        self.name_rest = name_rest

File: proyect_rappi/Java/Restaurant.java
class Restaurant extends Products{
    String type_food;
    Integer sell_by_date;
    String name_rest;
    public Restaurant(String name,String brand,Integer amount,String type_food,Integer sell_by_date,String name_rest){
        super(name,brand,amount);
        this.type_food = type_food;
        this.sell_by_date = sell_by_date;
        this.name_rest = name_rest;
    }
}
